examples/todomvc/index.js

```diff
@@ -5,12 +5,22 @@ import { Provider } from 'react-redux'
 import App from './containers/App'
 import configureStore from './store/configureStore'
 import 'todomvc-app-css/index.css'
+import { start, StateInfoConfig } from 'navigation'
+import configureStateInfo from './routes'
 
+configureStateInfo()
 const store = configureStore()
 
-render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
-  document.getElementById('root')
-)
+var todomvc = StateInfoConfig.dialogs.todomvc
+todomvc.states.app.navigated =  (data) => {
+  render(
+    <Provider store={store}>
+      <App filter={data.filter} />
+    </Provider>,
+    document.getElementById('root')
+  )
+}
```

Comment:
Any time the URL changes the Navigation router calls this navigated function which re-renders the App with the new filter